src/otg/gwas_catalog_study_curation.py

```diff
@@ -0,0 +1,70 @@
+"""Step to process GWAS Catalog associations and study table."""
```

Comment:
I'd also rename the module to `gwas_catalog_study_metadata_curation.py`
```suggestion
""""""Step to output a table with studies from the GWAS Catalog that require metadata curation or quality control.""""""
```